packages/contracts/yarn.lock

```diff
@@ -190,6 +190,21 @@ soroban-client@^0.2.1:
     stellar-base "8.0.1-soroban.6"
     urijs "^1.19.1"
```

Comment:
Can this get dropped? The old version shouldn't be around at all.

Reply:
what are you referring to ? `8.0.1-soroban.6` ? afaict, this is the latest, isn't it ?

Reply:
its autogen via yarn.